### PR TITLE
mixer: Adds bullet list in format bump; adds Example 3 for editors.

### PR DESCRIPTION
**Pull request by mvincerx**
@bktan8 We need to review this, and see also Ln 369 todo. 
Signed-off-by: Michael Vincerra <michael.vincerra@intel.com>

### CODE REVIEW

File: source/clear-linux/guides/tooling/mixer.rst [hunk 1]
```diff
@@ -339,12 +330,47 @@ set to get a smaller kernel image, which will also be faster to load.
...
+   .. code-block:: bash
+
+      mixer bundle list
+
+#. Add the `editors` bundle from upstream.
+
+   .. code-block:: bash
+
+      mixer bundle add editors
+
```
Comment (bktan8) [inline source/clear-linux/guides/tooling/mixer.rst]: Add step:
#. Make it a local bundle first before you can modify it.

.. code-block:: bash

   mixer bundle edit editors

File: source/clear-linux/guides/tooling/mixer.rst [hunk 2]
```diff
@@ -339,12 +330,47 @@ set to get a smaller kernel image, which will also be faster to load.
...
+#. Remove `joe` from its definition.
```
Comment (bktan8) [inline source/clear-linux/guides/tooling/mixer.rst]: Change this to: 

#. Use an editor and remove `joe` from the bundle definition.

.. code-block:: bash

   $EDITOR ./local-bundles/editors

File: source/clear-linux/guides/tooling/mixer.rst [hunk 3]
```diff
@@ -339,12 +330,47 @@ set to get a smaller kernel image, which will also be faster to load.
...
+
+   .. code-block:: bash
```
Comment (bktan8) [inline source/clear-linux/guides/tooling/mixer.rst]: Remove this line.

File: source/clear-linux/guides/tooling/mixer.rst [hunk 4]
```diff
@@ -339,12 +330,47 @@ set to get a smaller kernel image, which will also be faster to load.
...
+
+      mixer bundle edit editors
```
Comment (bktan8) [inline source/clear-linux/guides/tooling/mixer.rst]: Remove this line.

File: source/clear-linux/guides/tooling/mixer.rst [hunk 5]
```diff
@@ -339,12 +330,47 @@ set to get a smaller kernel image, which will also be faster to load.
    Look in ~/mixer/update/www/<mix version> to see the update content in your
    workspace.
 
-Example 3: Deploy updates to target
+Example 3: Edit bundle content
```
Comment (bktan8) [inline source/clear-linux/guides/tooling/mixer.rst]: I think "Example 3" is good to have, but it's in an odd location.  Maybe, it should be placed after "Deploy Updates to Target".

File: source/clear-linux/guides/tooling/mixer.rst (hunk 3 above)
Comment (mvincerx) [reply source/clear-linux/guides/tooling/mixer.rst]: Please verify the new sequence of steps.  Some of your comments overlapped with each other.

File: source/clear-linux/guides/tooling/mixer.rst [hunk 6]
```diff
@@ -244,6 +244,28 @@ set to get a smaller kernel image, which will also be faster to load.
       mixer bundle remove kernel-native
       mixer bundle add kernel-kvm
 
+#. In this case, we will add the `editors` bundle from upstream, but we will
+   remove the `joe` editor.
+
+   .. code-block:: bash
+
+      mixer bundle add editors
+
```
Comment (bktan8) [inline source/clear-linux/guides/tooling/mixer.rst]: Remove the empty line 253

File: source/clear-linux/guides/tooling/mixer.rst [hunk 7]
```diff
@@ -378,21 +394,28 @@ mix version 10 (from which the image was built), to mix version 20.
...
       swupd bundle-list -a
 
-   .. note::
+#. Now we will add the `editors` bundle that we modified.
+
+   .. code-block:: bash
+
+      swupd bundle add editors
+
+#. Try to start `joe` editor.  It should not appear because we removed it
```
Comment (bktan8) [inline source/clear-linux/guides/tooling/mixer.rst]: Pls add "the" after "start" as in "Try to start the `joe` editor."

File: source/clear-linux/guides/tooling/mixer.rst [hunk 8]
```diff
@@ -378,21 +394,28 @@ mix version 10 (from which the image was built), to mix version 20.
...
+   from the original `editors` bundle.
 
-      You cannot see the curl bundle that you added in Example 2 because
-      your mix is still on version 10.
+#. Next we will update from version 10 to 20 to capture the newly
+   available bundles.
 
-   Check for updates. You should see that version 20 is available. Use
-   :command:`swupd` to update your mix:
+#. Check to see that version 20 is available. Use :command:`swupd`
```
Comment (bktan8) [inline source/clear-linux/guides/tooling/mixer.rst]: I think line 406 and 409 are saying the same thing.  I think you should remove 409 and just put the code-block under under line 406.

Comment (mvincerx) [reply source/clear-linux/guides/tooling/mixer.rst]: Done

**Comment by mvincerx** [comment]
Done.

**Pull request merged**